terra/compute/base.py

```diff
@@ -300,13 +302,39 @@ def configure_logger(sender, **kwargs):
         sender._log_file = os.devnull
       if settings.processing_dir:
         os.makedirs(settings.processing_dir, exist_ok=True)
+
+      if settings.logging.server.family == "AF_UNIX":
+        # import pdb; pdb.set_trace()
+        socket_dir = Path(settings.logging.server.listen_address).parent
```

Comment:
Should there be a compatibility check for `server.family` and `listen_address`?  For example, would the following settings cause some issues:
```
        "server": {
          "port": 1234,
          "listen_host": "localhost",
          "listen_address": ["localhost", 1234],
          "family": "AF_UNIX"
        },
```